AppSettingsManagerUI/src/app/login/login.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="login-container">
```

Comment:
I would recommend structuring the elements better. Instead of just using divs, use appropriate HTML elements depending on your use case. Elements such as headers and sections can really improve the structure of our UI.

One more suggestion is to add form validation since Angular has built validators. This will ensure that the user enters valid data in the required fields.

Reply:
Yeah if you can lend some better HTML expertise than what I have that would be great. HTML/CSS is definitely my weakest area of knowledge